### PR TITLE
Enter/show/edit email recipients in the source modal and table

### PR DESCRIPTION
**Pull request by axmb**
email

### CODE REVIEW

File: verification/curator-service/ui/src/components/AutomatedSourceForm.tsx [hunk 1]
```diff
@@ -181,6 +200,29 @@ export default function AutomatedSourceForm(props: Props): JSX.Element {
...
+                                        data-testid="recipients"
+                                        helperText={
+                                            errors.notificationRecipients
+                                                ? 'Values must be valid email addresses'
+                                                : undefined
+                                        }
+                                        fullWidth
+                                        alwaysShowPlaceholder
+                                        placeholder="Notification recipient emails"
+                                        defaultValue={[props.user.email]}
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/AutomatedSourceForm.tsx:217]: Would be better to have this as initialValues.notificationRecipients so it's only defined in one place

File: verification/curator-service/ui/src/components/SourceTable.tsx [hunk 2]
```diff
@@ -93,6 +95,8 @@ interface TableRow {
     awsScheduleExpression?: string;
     // dateFilter
     dateFilter?: DateFilter;
+    // notificationRecipients
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/SourceTable.tsx:98]: Comment doesn't add much value :) remove? or describe in other words what it is

File: verification/curator-service/ui/src/components/SourceTable.tsx [hunk 3]
```diff
@@ -361,6 +366,29 @@ class SourceTable extends React.Component<Props, SourceTableState> {
...
+                                    rowData.notificationRecipients
+                                        ? rowData.notificationRecipients?.join(
+                                              '\n',
+                                          )
+                                        : '',
+                                editComponent: (props): JSX.Element => (
+                                    <ChipInput
+                                        defaultValue={props.value || []}
+                                        // material-ui-chip-input doesn't define this type.
+                                        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/SourceTable.tsx:384]: Can you get rid of this now that there isn't an `any` there?

**Review by allysonjp715 — APPROVED** [review]

File: verification/curator-service/ui/src/components/SourceTable.tsx (hunk 2 above)
Comment (axmb) [reply verification/curator-service/ui/src/components/SourceTable.tsx:98]: Yeah, axed a couple of these.

File: verification/curator-service/ui/src/components/SourceTable.tsx (hunk 3 above)
Comment (axmb) [reply verification/curator-service/ui/src/components/SourceTable.tsx:384]: Done, thanks.

File: verification/curator-service/ui/src/components/AutomatedSourceForm.tsx (hunk 1 above)
Comment (axmb) [reply verification/curator-service/ui/src/components/AutomatedSourceForm.tsx:217]: SG, done.

**Pull request merged**